src/data/committeeMembers.ts

```diff
@@ -137,4 +138,10 @@ export const committeeMembers = [
     role: "Logistics & Facility",
     profileImageUrl: AshutoshImage.src,
   },
+  {
+    name: "Roshan Sahani",
+    team: "Local team",
+    role: "Finances",
```

Comment:
```suggestion
    role: "Sponsorship & Finances",
```
https://docs.ubucon.asia/organizer/teams-roles/